src/Raven.Server/Documents/ETL/Providers/Elasticsearch/ElasticsearchEtl.cs

```diff
@@ -0,0 +1,160 @@
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertAttachmentTombstonesEnumerator(DocumentsOperationContext context, IEnumerator<Tombstone> tombstones,
+            List<string> collections)
+        {
+            throw new NotSupportedException("Attachment tombstones aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertCountersEnumerator(DocumentsOperationContext context, IEnumerator<CounterGroupDetail> counters,
+            string collection)
+        {
+            throw new NotSupportedException("Counters aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertTimeSeriesEnumerator(DocumentsOperationContext context, IEnumerator<TimeSeriesSegmentEntry> timeSeries,
+            string collection)
+        {
+            throw new NotSupportedException("Time series aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override IEnumerator<ElasticsearchItem> ConvertTimeSeriesDeletedRangeEnumerator(DocumentsOperationContext context,
+            IEnumerator<TimeSeriesDeletedRangeItem> timeSeries, string collection)
+        {
+            throw new NotSupportedException("Time series aren't supported by ELASTICSEARCH ETL");
+        }
+
+        protected override EtlTransformer<ElasticsearchItem, ElasticsearchIndexWithRecords, EtlStatsScope, EtlPerformanceOperation> GetTransformer(DocumentsOperationContext context)
+        {
+            return new ElasticsearchDocumentTransformer(Transformation, Database, context, Configuration);
+        }
+
+        protected override int LoadInternal(IEnumerable<ElasticsearchIndexWithRecords> records, DocumentsOperationContext context, EtlStatsScope scope)
+        {
+            Uri[] nodes = Configuration.Connection.Nodes.Select(x => new Uri(x)).ToArray();
+            var pool = new StaticConnectionPool(nodes);
+            var settings = new ConnectionSettings(pool);
+            var client = new ElasticClient(settings);
+            int statsCounter = 0;
+            
+            foreach (var index in records)
+            {
+                StringBuilder deleteQuery = new StringBuilder();
+                
+                foreach (ElasticsearchItem delete in index.Deletes)
+                {
+                    deleteQuery.Append($"{delete.DocumentId},");
+                }
+
+                var deleteResponse = client.DeleteByQuery<string>(d => d
+                    .Index(index.IndexName.ToLower())
+                    .Query(q => q
+                        .Match(p => p
+                            .Field(index.IndexIdProperty)
+                            .Query($"{deleteQuery}"))
+                    )
+                );
+
+                if (deleteResponse.ServerError != null)
+                {
+                    throw new Exception(deleteResponse.ServerError.Error.ToString());
```

Comment:
Can we have a test were we put _incorrect_ URLs of Elasticsearch and see the this generates an alert about load error?

See: https://github.com/ravendb/ravendb/blob/e36526d2c4c8a13fecf6dcea2c4dd47a53c3c8e7/src/Raven.Server/NotificationCenter/EtlNotifications.cs#L34-L43